documentation/src/main/asciidoc/topics/proc_configuring_pom_xml.adoc

```diff
@@ -0,0 +1,49 @@
+[id='configure_pom']
+= Configuring Your {brandname} POM
+Maven uses configuration files called Project Object Model (POM) files to
+define projects and manage builds. POM files are in XML format and describe the
+module and component dependencies, build order, and targets for the resulting
+project packaging and output.
+
+.Procedure
+
+. Open your project `pom.xml` for editing.
+. Define the `version.infinispan` property with the correct {brandname} version.
+. Include the `infinispan-bom` in a `dependencyManagement` section.
++
+The Bill Of Materials (BOM) controls dependency versions, which avoids version
+conflicts and means you do not need to set the version for each {brandname}
+artifact you add as a dependency to your project.
++
+. Add {brandname} artifacts in a `dependencies` section as required.
+. Save and close `pom.xml`.
+
+The following example shows the {brandname} version, BOM, and dependencies:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<properties>
+  <version.infinispan>{mvn_version}</version.infinispan>
```

Comment:
where is this set?
can you update `ispn_bom.xml` to include this info (with the upstream version)? There is no difference between upstream/downstream except the RH GA repository.

Reply:
that is set here: https://gitlab.cee.redhat.com/red-hat-jboss-data-grid-documentation/rhdg-documentation/merge_requests/370/diffs#0bac8f4ce17e78071c1b5d6e841b363ecf9ea8fe_1_19

it's manual for now. I get the version from the JDG repo but I plan to script it and do a bit of awk/sed magic to automate it via jenkins.

isn't the version different for community though? you get the coordinates and version from the download page: https://infinispan.org/download/

Reply:
the versions are different. but if we can keep `ispn_bom.xml` example only that would be great IMO...